ui/components/app/edit-gas-fee-popover/edit-gas-item/edit-gas-item.js

```diff
@@ -64,7 +64,8 @@ const EditGasItem = ({ priorityLevel }) => {
 
   if (
     priorityLevel === PRIORITY_LEVELS.DAPP_SUGGESTED &&
-    !dappSuggestedGasFees
```

Comment:
What do you think about adding a test for when `dappSuggestedGasFees` is present, but neither `dappSuggestedGasFees.maxFeePerGas` nor `dappSuggestedGasFees.gasPrice` is present?

Reply:
added 👍